lib/view_component/slotable_v2.rb

```diff
@@ -142,6 +142,23 @@ def renders_many(slot_name, callable = nil)
         register_slot(slot_name, collection: true, callable: callable)
       end
 
+      def renders_one?(slot_name)
+        registered_slots.dig(slot_name, :collection) == false
+      end
+
+      def renders_many?(slot_name)
+        registered_slots.dig(slot_name, :collection) == true
+      end
+
+      def renders_many_item?(slot_name)
+        plural_slot_name = ActiveSupport::Inflector.pluralize(slot_name).to_sym
+        plural_slot_name != slot_name && renders_many?(plural_slot_name)
+      end
+
+      def renders_slot?(slot_name)
```

Comment:
@jonspalmer based on the context provided, do you need all of these methods, or just `renders_slot`? If it's just the latter, I think we could inline the other methods into this one to avoid the naming question for now.

Reply:
I definitely need a way to get the type of the slot back. Important to be able to distinguish between a "one" slot and a singular setter of a many slot. So either three methods or perhaps just this

```ruby
SlotsV2Component.render_slot?(slot_name) #=>  :one, :many, :many_item, nil
```

The only naming issue there is `:many_item` Got any  better ideas for that?

Once we settle on the names I can refactor the PR>

Reply:
Gotcha. How about:

```ruby
SlotsV2Component.slot_type(slot_name) #=>  :single, :collection, :collection_item, nil
```

Since we're returning a symbol or nil, I think a regular method name vs. a predicate is more appropriate.

Reply:
Sold. I will make it so